### DISCUSSION TITLE
List of all supported keyboard shortcuts

### DISCUSSION THREAD

**Discussion by mike2307** [blakeblackshear/frigate#15392] [Ask A Question] [answered]
Hi all,

when browsing through the docs I noticed a nice list of keyboard shortcuts when annotating images in the frigate+ editor: https://docs.frigate.video/plus/improving_model/#shortcuts-for-a-faster-workflow

Is there a similar list anywhere for frigate itself?
I know that one can select all items in the explore windows by using Ctrl+A.
But I have no idea how I can select a couple of events manually (e.g. for deletion).

Thanks a lot in advance.

**Comment by hawkeye217** [comment, marked as answer]
We'll be posting this with the Frigate 0.15 official release. For now, you can look at Frigate's frontend code for the `useKeyboardListener` hook to see what keys are available.

https://github.com/search?q=repo%3Ablakeblackshear%2Ffrigate%20useKeyboardListener&type=code
